packages/jest/src/index.js

```diff
@@ -118,10 +118,9 @@ module.exports = (neutrino, opts = {}) => {
 
       writeFileSync(configFile, `${JSON.stringify(options, null, 2)}\n`);
 
-      jest.runCLI(cliOptions, options.roots || [options.rootDir], result =>
-        (result.numFailedTests || result.numFailedTestSuites ?
-          reject() :
-          resolve()));
+      jest.runCLI(cliOptions, options.roots || [options.rootDir])
```

Comment:
I was just thinking, since we are no longer using a callback here, we can remove the `return new Promise` on L99 and instead just return the `runCLI` call, right?

```js
return jest.runCLI(cliOptions, options.roots || [options.rootDir]);
```

Do we even then need to capture `results.success`, or will jest reject if `success` is false?

Reply:
It won't reject as far as I can say by looking at their code. I think their promise is for some unexpected inside a `try ... catch`. That's why they have the `success` property.

Reply:
OK, so we will need to manually throw or reject or whatever. Gotcha.

Reply:
Yeah. The implementation I've submitted in this PR works in my repository. I dunno why are tests failing, though.

Reply:
OK, I'll see if I can clone locally and replicate.

Reply:
If you make this change, I will get it out in a patch release today. :)

Reply:
🤔 why did it work for me? Did they break the API again or I’m just stupid? I’ll test the code again with different Jest versions to check that.

Reply:
So, the just tests pass, the only thing was we were using the results properties from jest to determine pass or fail incorrectly, hence why it was failing our tests.

Reply:
No, I mean I tried this fix on my project and it fixed the issue. That's why I submitted the fix in the first place.

Reply:
Hmm not sure. You did fix the fact that it should be a promise now, so maybe that was part of it.